cpp/tests/io/parquet_test.cpp

```diff
@@ -716,6 +716,59 @@ TEST_F(ParquetWriterTest, Strings)
   cudf::test::expect_metadata_equal(expected_metadata, result.metadata);
 }
 
+TEST_F(ParquetWriterTest, StringsAsBinary)
+{
+  std::vector<const char*> unicode_strings{
+    "Monday", "Wȅdnȅsday", "Friday", "Monday", "Friday", "Friday", "Friday", "Funday"};
+  std::vector<const char*> ascii_strings{
+    "Monday", "Wednesday", "Friday", "Monday", "Friday", "Friday", "Friday", "Funday"};
+
+  column_wrapper<cudf::string_view> col0{ascii_strings.begin(), ascii_strings.end()};
+  column_wrapper<cudf::string_view> col1{unicode_strings.begin(), unicode_strings.end()};
+  column_wrapper<cudf::string_view> col2{ascii_strings.begin(), ascii_strings.end()};
+  cudf::test::lists_column_wrapper<int8_t> col3{{'M', 'o', 'n', 'd', 'a', 'y'},
+                                                {'W', 'e', 'd', 'n', 'e', 's', 'd', 'a', 'y'},
+                                                {'F', 'r', 'i', 'd', 'a', 'y'},
+                                                {'M', 'o', 'n', 'd', 'a', 'y'},
+                                                {'F', 'r', 'i', 'd', 'a', 'y'},
+                                                {'F', 'r', 'i', 'd', 'a', 'y'},
+                                                {'F', 'r', 'i', 'd', 'a', 'y'},
+                                                {'F', 'u', 'n', 'd', 'a', 'y'}};
+
+  std::vector<std::unique_ptr<column>> cols;
+  cols.push_back(col0.release());
+  cols.push_back(col1.release());
+  cols.push_back(col2.release());
+  cols.push_back(col3.release());
+  auto write_tbl = std::make_unique<table>(std::move(cols));
+  EXPECT_EQ(4, write_tbl->num_columns());
+
+  cudf_io::table_input_metadata expected_metadata(*write_tbl);
+  expected_metadata.column_metadata[0].set_name("col_single").set_output_as_binary(true);
```

Comment:
I'm not sure I caught if non-ASCII contents may be configured to `set_output_as_binary()`. 
In this test, both columns that are set to write binary also happen to have ASCII contents.

Reply:
output as binary is only a valid thing to set on a string column with this PR. Next PR ups that to include list<int8>. I don't have negative tests here though, where the user incorrectly identifies a column as output as binary.

Reply:
Sorry for the confusion. My point was whether to test with `.set_output_as_binary(true)` for `col1`.

Reply:
oh, so I found out in testing, that I didn't know an easy way to generate list<int8> columns with unicode in them, but THAT IS NO EXCUSE! I've figured out the unicode characters and added that to the test as well.